docs/src/examples/components/Chat/Content/ChatExampleReactionGroupMeReacting.shorthand.tsx

```diff
@@ -1,12 +1,12 @@
 import * as React from 'react'
-import { Avatar, Chat } from '@stardust-ui/react'
+import { Avatar, Chat, ChatItemProps, ShorthandCollection } from '@stardust-ui/react'
 
 const reactions = [
```

Comment:
Let's type the reactions as well. Wondering why this wasn't catch... :(

Reply:
Done 👍